backend/src/data/repositories/master/master.repository.ts

```diff
@@ -23,7 +30,7 @@ class Master {
       return null;
     }
 
-    return Master.modelToEntity(master);
+    return Master.modelToEntity(master, []);
```

Comment:
Here you also need to fetch permissions (similarly to the getById)

Reply:
+